src/test/java/com/openlattice/rehearsal/search/SearchControllerTest.kt

```diff
@@ -410,6 +421,7 @@ class SearchControllerTest : MultipleAuthenticatedUsersBase() {
 
         /* Soft delete */
         dataApi.deleteAllEntitiesFromEntitySet(es2.id, DeleteType.Soft)
+        Thread.sleep(1000)
```

Comment:
Seems like a lot of thread.sleep in here but as long as it doesn't take too long on Bamboo, it's cool with me

Reply:
Yeah, locally it wouldn't index before the search call if I didn't put a sleep there, but may be would run on Bamboo without it